sunpy/net/helioviewer.py

```diff
@@ -23,42 +23,75 @@
 
 class HelioviewerClient(object):
     """Helioviewer.org Client"""
-    def __init__(self, url="https://legacy.helioviewer.org/api/"):
+
+    datasource_info = dict()
```

Comment:
This doesn't need to be in `__init__`? or `self.`?

Reply:
Nope. It's working perfectly fine. We just created a global variable

Reply:
That's fine but I think this should be moved into `__init__` just be safe.